README.md

```diff
@@ -142,6 +143,10 @@ A curated list of awesome things related to <a href='https://github.com/vitejs/v
 - [vite-plugin-mocker](https://github.com/minjs1cn/vite-plugin-mocker) - Mocker server.
 - [vite-plugin-test](https://github.com/aelbore/vite-plugin-test) - Headless testing your component.
 
+#### Dev Servers
+
+- [vite-plugin-node](https://github.com/axe-me/vite-plugin-node) - Use Vite as node dev server.
```

Comment:
The plugin looks good to me. But it took me a while to understand what's it trying to solve. Can you update the description to be more clear and accurate? Thanks.

Reply:
wording is hard for programmer lol

Reply:
Integration with Node.js backend servers?

Reply:
thx, updated.